RevenueCatUI/Data/LocalizedAlertError.swift

```diff
@@ -18,13 +18,13 @@ public struct LocalizedAlertError: LocalizedError {
     public let errorDescription: String?
     public let failureReason: String?
     public let recoverySuggestion: String?
-    
+
     public init(errorDescription: String?, failureReason: String?, recoverySuggestion: String?) {
         self.errorDescription = errorDescription
         self.failureReason = failureReason
         self.recoverySuggestion = recoverySuggestion
     }
-    
+
```

Comment:
this is what they have changed:
```
-    var failureReason: String? {
-        if let errorCode = self.underlyingError as? ErrorCode {
-            return "Error \(self.underlyingError.code): \(errorCode.description)"
-        } else {
-            return self.underlyingError.localizedDescription
```

so should we.
e.g. 
```
        case is ErrorCode:
           "Error \(self.underlyingError.code): \(errorCode.description)"
        default:
            error.localizedDescription
        }
```

Reply:
Changed 👍